wpilibj/src/main/java/edu/wpi/first/wpilibj/OnboardIMU.java

```diff
@@ -159,7 +159,19 @@ public double getAccelZ() {
 
   private double[] getRawEulerAngles() {
     double[] anglesRaw = new double[3];
-    IMUJNI.getIMUEulerAngles(anglesRaw);
+    switch (m_mountOrientation) {
+      case kFlat:
+        IMUJNI.getIMUEulerAnglesFlat(anglesRaw);
+        break;
+      case kLandscape:
+        IMUJNI.getIMUEulerAnglesLandscape(anglesRaw);
+        break;
+      case kPortrait:
+        IMUJNI.getIMUEulerAnglesPortrait(anglesRaw);
+        break;
+      default:
+        break;
```

Comment:
```suggestion
      case kFlat -> IMUJNI.getIMUEulerAnglesFlat(anglesRaw);
      case kLandscape -> IMUJNI.getIMUEulerAnglesLandscape(anglesRaw);
      case kPortrait -> IMUJNI.getIMUEulerAnglesPortrait(anglesRaw);
```
We switched to using the new Java switch syntax in #6691, though it does look like `getYawNoOffset()` also uses the old syntax, so we could just hit this and `getYawNoOffset()` in the same PR later.